sky/serve/autoscalers.py

```diff
@@ -85,7 +95,9 @@ def update_version(self, version: int,
                              else spec.min_replicas)
         # Reclip self.target_num_replicas with new min and max replicas.
         self.target_num_replicas = max(
-            self.min_replicas, min(self.max_replicas, self.target_num_replicas))
+            self.min_replicas,
+            min(self.max_replicas, (spec.init_replicas if spec.init_replicas
+                                    is not None else spec.min_replicas)))
```

Comment:
Do we really need this here? seems like we'll update the Ntar in subclass' implementation

Reply:
removed

Reply:
We don't really need this, but we can leave it here in case other subclasses (not `RequestRateAutoscaler`) might call this